tests/test_mathematical.py

```diff
@@ -137,16 +137,22 @@ def _test_mathematical(self, name, dtype, lhs, rhs):
         else:
             result = getattr(dpnp, name)(a_dpnp, b_dpnp)
             expected = getattr(numpy, name)(a_np, b_np)
-            assert_allclose(result, expected, rtol=1e-6)
+            if numpy.issubdtype(expected.dtype, numpy.floating):
```

Comment:
isn't the same can be done through `assert_dtype_allclose`?

Reply:
I was not aware of it. thanks